capi/README.md

```diff
@@ -1,3 +0,0 @@
-# capi cookbook
```

Comment:
There's already a README.md on the level above. So I could either delete it, or merge its content into the one there. I felt like this information is not really useful, because we already know that a sprout contains recipes. Therefore deleted it. Or was it meant as a joke? :-)